lib/lob/v1/client.rb

```diff
@@ -0,0 +1,68 @@
+# Dynamically require files
+Dir[File.join(File.dirname(__FILE__), '*.rb')].each {|file| require_relative file }
```

Comment:
I wouldn't be opposed to just listing out all of the requirements here, since they have to be defined below anyway. I'll leave it up to you though.